plugins/dataTypes/List/lang/nl.php

```diff
@@ -27,10 +27,10 @@
 $L["help"] = "Voer een lijst in met items, gescheiden door een pipe | karakter. Selecteer vervolgens of u wilt <b>Precies</b> X aantal items, of <b>Bij de meeste</b> X items uit de lijst. Meerdere items worden geretourneerd in een door komma's gescheiden lijst in de resultaten. Als u wilt dat uw gegevens ook lege waarden mogen bevatten, voeg dan een of meer pipe tekens aan het eind in - des te meer pipe tekens die u invoert, des te groter de kans dat een lege waarde wordt gegenereerd.";
 $L["incomplete_fields"] = "De Aangepaste lijst Gegevenstype dient te beschikken over het formaat ingevoerd in het Opties tekstveld. Please fix de volgende rijen:";
 $L["invalid_int_fields"] = "Gelieve enkel getallen in te voeren in de \"Precies\" en \"maximaal\" velden voor de Aangepaste lijst Gegevenstype. Pas AUB de volgende rijen aan:";
-$L["one_to_ten"] = "een | twee | drie | vier | vijf | zes | zeven | acht | negen | tien";
-$L["prefix"] = "Dr. | De heer | Mevrouw | Ms | Prof. | Drs. | Mr. | Ir. | Ing.";
-$L["relationship_states"] = "Alleenstaand | Getrouwd | Gescheiden | Samenwonend";
+$L["one_to_ten"] = "een|twee|drie|vier|vijf|zes|zeven|acht|negen|tien";
+$L["prefix"] = "Dr.|De heer|Mevrouw|Ms|Prof.|Drs.|Mr.|Ir.|Ing.";
+$L["relationship_states"] = "Alleenstaand|Getrouwd|Gescheiden|Samenwonend";
 $L["separated_by_pipe"] = "Voer waarden in gescheiden door |";
```

Comment:
Thanks for these updates! One request: in future, try to keep a PR specific to a single thing. This PR is all about the Boolean Data Type, so any changes to List would be a separate PR. No big deal - just makes it easier to keep track of that way.